docs/build/bridge/index.md

```diff
@@ -46,7 +46,7 @@ In the bash snippet below we use `jq` to parse a JSON output. You can choose to
 Otherwise, follow instructions here to install in your distribution: https://stedolan.github.io/jq/.
 
 ## Set environment variables
-**Please reach out to us on [Slack](https://centrifuge.io/slack) or [Telegram](https://t.me/centrifuge_chat) for contract addresses.**
+**Please reach out to us on [Discord](https://discord.gg/59E2h2pAtb) or [Telegram](https://t.me/centrifuge_chat) for contract addresses.**
```

Comment:
Does it make sense to ask people to reach out to us on Telegram (https://t.me/centrifuge_chat), given it's a read only channel now?

Reply:
@Rhanok09 Usually TG is a good way to communicate to an admin about the security vulnerability. I would definitely leave it there, but let`s what others decide!

Reply:
I'd disagree that linking to TG serves a meaningful purpose but if people want to leave it there, it's fine!